man/mungepiece.Rd

```diff
@@ -4,7 +4,7 @@
 \name{mungepiece}
 \alias{mungepiece}
 \title{Mungepiece.}
-\format{\preformatted{Class 'R6ClassGenerator' <environment: 0x7fd6890c2d58> 
+\format{\preformatted{Class 'R6ClassGenerator' <environment: 0x7fd6897d8148> 
```

Comment:
Yeah haven't gotten there yet, this is an easy fix

On Mon, Oct 5, 2015 at 12:33 AM, Kirill Sevastyanenko <
notifications@github.com> wrote:

> In man/mungepiece.Rd
> <https://github.com/robertzk/mungebits2/pull/12#discussion_r41111836>:
>
> > @@ -4,7 +4,7 @@
> >  \name{mungepiece}
> >  \alias{mungepiece}
> >  \title{Mungepiece.}
> > -\format{\preformatted{Class 'R6ClassGenerator' <environment: 0x7fd6890c2d58>
> > +\format{\preformatted{Class 'R6ClassGenerator' <environment: 0x7fd6897d8148>
>
> probably need a dontrun
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robertzk/mungebits2/pull/12/files#r41111836>.
>



-- 
-Robert Krzyzanowski